app/scripts/migrations/081.ts

```diff
@@ -0,0 +1,162 @@
+import { cloneDeep, isArray } from 'lodash';
+import {
+  WALLET_SNAP_PERMISSION_KEY,
+  SnapCaveatType,
+} from '@metamask/rpc-methods';
```

Comment:
This brings in snaps dependencies to the beta and main extension. We should be sure that that is fine.

Reply:
If not, we could hardcode the values for now. Potentially they should be, because they should reflect the CaveatType and key when this migration was applied?

Reply:
> If not, we could hardcode the values for now. Potentially they should be, because they should reflect the CaveatType and key when this migration was applied?

Yes, it should reflect the extension code at the time. Aren't the packages brought in regardless? Only thing we do in beta, main is fencing no?

Reply:
The packages are only brought in to the build if they are required in a file. Could we swap it out for hardcoded strings instead then? So it matches the current version just in case we change them in the future

Reply:
okay cool, ill hardcode.